Resources/Private/Language/de.locallang_tca.xlf

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.0" xmlns:t3="http://typo3.org/schemas/xliff">
+    <file source-language="en" target-language="de" datatype="plaintext" original="messages" date="2018-06-20T11:17:23Z" product-name="marketing_automation">
+        <header />
+        <body>
+            <trans-unit id="tx_marketingautomation_persona">
+                <source>Persona</source>
+                <target>Persona</target>
```

Comment:
I would remove the source labels from translations files. It's not necessary and sources might get updated.